src/Servers/Connections.Abstractions/ref/Microsoft.AspNetCore.Connections.Abstractions.netcoreapp.cs

```diff
@@ -123,8 +123,58 @@ public partial interface IConnectionListenerFactory
     {
         System.Threading.Tasks.ValueTask<Microsoft.AspNetCore.Connections.IConnectionListener> BindAsync(System.Net.EndPoint endpoint, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken));
     }
-    public partial interface IMultiplexedConnectionListenerFactory : Microsoft.AspNetCore.Connections.IConnectionListenerFactory
+    public partial interface IMultiplexedConnectionBuilder
     {
+        System.IServiceProvider ApplicationServices { get; }
+        Microsoft.AspNetCore.Connections.MultiplexedConnectionDelegate BuildMultiplexed();
```

Comment:
Rename to Build the factory is already multiplexed, why do the method names need to be as well?

Reply:
This interface will be implemented by ListenOptions which already implements IConnectionBuilder. Build() can't be named the same thing.